### DISCUSSION TITLE
[On hold till Mainline] [ACU Apps - Gaming] App testings

### DISCUSSION THREAD

**Discussion by DarkevilPT** [kwankiu/acu#6] [General]
# ACU APPS
// Gaming

Pegasus FrontEnd / Works still downloading a game to try it but the app launches. Couldnt get the metadata written right for the game to work seems a bit wierd...
Maybe its just like the dolphin emulator (which I have no experience with) and probably works best in dolphin.

---

// Steam
![image](https://github.com/kwankiu/acu/assets/3583885/75fb5fb6-c1dc-464e-b888-4e4b76de9fa3)
Goes into an infinite terminal corefont bug and doenst starts steam with that error.
However it is installed somewhere and can be manipulated somehow and that needs testing aka try n error.

---

// Lutris 
Despite the installation looked like it failed..
![image](https://github.com/kwankiu/acu/assets/3583885/75270eae-e822-4a20-aebb-a93cccc2ddcb)
It does start.
I also dont.. have much experience with Lutris.. idk what can be done with it... doubt it replaces steam to its core but I assume that it can for epic and battle.net stores.

---

// Virtualization
Box64 --- broooo it takes... LONG is it doing Jnproc thing using all cores I mean?
(or maybe this is because its being reinstalled after I had installed steam)

Malior-Droid is a master piece.. works as always on 5.10 and.. its a shame that if this gets 'lost' we will lose this utility.

**Comment by kwankiu** [comment]
> Box64 --- broooo it takes... LONG is it doing Jnproc thing using all cores I mean?

Nope, it currently follows your system's makepkg's conf
To speed up, you can modify makepkg, boogie has mentioned how to do so in radxa's discord

In the future, ACU should add support to utilise tools like ARB or AGR, maybe even Paru and Yay

**Comment by DarkevilPT** [comment]
Just for suggestive 'induction' 
Cuttlefish Android alternative to Redroid seems to exist however malior droid was created to use mali blobs which does a better job but probably not going to work on mainline kernel while its shitly done.. (collabora or/and rockchip wise).

https://sites.google.com/junsun.net/how-to-run-cuttlefish/home#h.p_1tmL-CvpLe1P

https://gitcode.com/CNflysky/redroid-rk3588/overview?utm_source=csdn_github_accelerator

https://www.cnblogs.com/FW-ltlly/p/18014692

https://blog.bai.re/2023/02/23/cuttlefish-and-redroid-on-rock5b/

Things to try maybe..

**Comment by DarkevilPT** [comment]
> ACU should add support

On steam installation avoid asking for password :D and... somehow when Im trying to type it / it fails as if it had changed my password.. its strange.. but usually I forget this is installing due to the box compilations and when I came back to type the password it fails everytime.

![image](https://github.com/kwankiu/acu/assets/3583885/fe79ea13-d4c6-4717-b7ba-3a72ba87f8d8)

Fails every god dam time.. idk what makes this happen.

**Comment by DarkevilPT** [comment]
![Screenshot from 2024-06-09 19-57-31](https://github.com/kwankiu/acu/assets/3583885/b15fbe7c-24cd-46a8-93e3-43fef7bd440f)
Can you please fix this -y lacking options? Just so it installs?

**Comment by DarkevilPT** [comment]
I was looking for apps to check my temperatures and added a few and this happened.. didnt completed the install and the terminal flopped to read only out of the blue.

![image](https://github.com/kwankiu/acu/assets/3583885/1e2082d1-ae50-4573-93ef-03a8aeea409e)

Not completing the install

**Comment by DarkevilPT** [comment]
Overall I think these should all be reviewed once we get the mainline completed and.. maybe use the flatpak approach.
Even flatpak should be added to the ACU other than maybe us adding useless APPS unless exclusive to ACU, such as Malior Droid or a redroid format that can work.
